parlai/zoo/tutorial_transformer_generator/build.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Pretrained model allowing to get the same performances as in
+https://arxiv.org/abs/1905.01969.
```

Comment:
nit: think you linked the wrong paper? (this is polyencoder)

Reply:
yup